src/Engines/nativeEngine.ts

```diff
@@ -273,6 +273,20 @@ export class NativeEngine extends Engine {
 
         Tools.Log("Babylon Native (v" + Engine.Version + ") launched");
 
+        Tools.LoadScript = function (scriptUrl, onSuccess, onError, scriptId) {
+            Tools.LoadFile(scriptUrl, (data) => {
+                Function(data as string).apply(null);

```

Comment:
That's a great wai to parse a script string and execute it :-)

You can ask for an optional context (`this`) and replace it with the null, if provided.